packages/cmrjs/tests/cmr-utils/test-cmr-utils.js

```diff
@@ -1352,3 +1357,37 @@ test('buildCMRQuery works with if the input results list is empty', (t) => {
   const actual = buildCMRQuery(results);
   t.deepEqual(actual, expected);
 });
+
+test('setECHO10Collection updates echo10 collection name and version', async (t) => {
+  const filename = 'tests/cmr-utils/data/meta.xml';
+  const cmrObject = await promisify(xml2js.parseString)(fs.readFileSync(filename, 'utf-8'), xmlParseOptions);
+  const updated = setECHO10Collection(
+    cmrObject,
+    { name: 'a', version: 'b' }
+  );
+  t.is(updated.Granule.Collection.ShortName, 'a');
+  t.is(updated.Granule.Collection.VersionId, 'b');
+});
+
+test('updateCmrFileCollections updates Echo10Files when missing', (t) => {
```

Comment:
Change test title

Reply:
Also if there's a granule flag but no collection, it's still writing that right?    Add test coverage if accurate.